video/filter/vf_format.c

```diff
@@ -220,7 +227,8 @@ static const m_option_t vf_opts_fields[] = {
     {"dh", OPT_INT(dh)},
     {"dar", OPT_DOUBLE(dar)},
     {"convert", OPT_BOOL(convert)},
-    {"dolbyvision", OPT_BOOL(dovi)},
+    {"dynamic-hdr-metadata", OPT_BOOL(dynamic_hdr_metadata)},
+    {"dolbyvision", OPT_REPLACED("dynamic-hdr-metadata")},
```

Comment:
Wouldn' it be better to have separate "dolbyvision" and "hdr10+" suboptions? What if I want to strip dolby vision metadata but preserve dynamic HDR10+

Reply:
Why?

Reply:
Since you didn't answer, I've added new option for this. Still think it is too verbose for users who in practice want "enable peak-detection" instead of using dynamic metadata, whether it is dovi or hdr10+ is just a marketing detail.